starters/default/package-lock.json

```diff
@@ -10,16 +10,14 @@
       "license": "0BSD",
       "dependencies": {
         "gatsby": "^5.3.3",
-        "gatsby-plugin-image": "^3.3.2",
-        "gatsby-plugin-manifest": "^5.3.1",
-        "gatsby-plugin-sharp": "^5.3.2",
-        "gatsby-source-filesystem": "^5.3.1",
-        "gatsby-transformer-sharp": "^5.3.1",
         "react": "^18.2.0",
         "react-dom": "^18.2.0"
       },
       "devDependencies": {
-        "prettier": "^2.8.0"
+        "@types/node": "^18.11.10",
+        "@types/react": "^18.0.26",
+        "@types/react-dom": "^18.0.9",
+        "typescript": "^4.9.3"
       }
     },
```

Comment:
This doesn't look right list of deps definitely doesn't match deps in package.json for this starter - https://github.com/gatsbyjs/gatsby/blob/250c8b799a88bc07c5cef18b66dc8ebe2cd1ad9e/starters/default/package.json#L7-L19

Reply:
Ok, you are generally right about lock file being out of sync, just this PR changes to wrong one :)

I will be updating lock files in https://github.com/gatsbyjs/gatsby/pull/37331